packages/nux/src/components/dot-tip/test/__snapshots__/index.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`DotTip should render correctly 1`] = `
   aria-label="Editor tips"
   class="components-popover nux-dot-tip"
   role="dialog"
-  style="position: absolute; opacity: 0; transform: translateX(-2em) scale(0) translateZ(0); transform-origin: 0% 50% 0; left: 0px; top: 0px;"
+  style="position: absolute; opacity: 0.9994144868675858; transform: translateX(-0.00117em) scale(0.9994144868675858) translateZ(0); transform-origin: 0% 50% 0; left: 0px; top: 0px;"
```

Comment:
@jsnajdr's [comment](https://github.com/WordPress/gutenberg/pull/46227#discussion_r1037138303) (opacity value) applies here too and we should have the same handling.